### ISSUE TITLE
Cannot use with Logback

### ISSUE THREAD

**Issue by CommanderTvis** [VKCOM/vk-java-sdk#185]
I can't use the SDK with Logback+Slf4J, because SDK uses Log4j internally...
```
Exception in thread "main" java.lang.NoClassDefFoundError: org/apache/logging/log4j/LogManager
[22.12.2019 17:06:07] [org.eclipse.jetty.util.log] [Token Acceptor/DEBUG] Logging to Logger[org.eclipse.jetty.util.log] via org.eclipse.jetty.util.log.Slf4jLog
        at com.vk.api.sdk.httpclient.HttpTransportClient.<clinit>(HttpTransportClient.java:40)
        at io.github.commandertvis.vkimd.LongPoller.<clinit>(LongPoller.kt:13)
        at io.github.commandertvis.vkimd.VKIMD.main(VKIMD.kt:28)
Caused by: java.lang.ClassNotFoundException: org.apache.logging.log4j.LogManager
        at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
        ... 3 more
```

**Comment by CommanderTvis** [comment]
It would be much better if HttpTransportManager used Logger org.slf4j.LoggerFactory instead of org.apache.logging.log4j.LogManager: 

```java
public class HttpTransportClient implements TransportClient {

    private static final Logger LOG = LogManager.getLogger(HttpTransportClient.class);
```
->
```java
public class HttpTransportClient implements TransportClient {

    private static final Logger LOG = LoggerFactory.getLogger(HttpTransportClient.class);
```

**Comment by CommanderTvis** [comment]
The current possible workaround without forking the SDK is excluding log4j from classpath, then adding log4j-over-slf4j instead of it.

**Comment by aotd1** [comment]
Thanks, fixed in 1.0.8, please check it
